dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.wsSecurity1.1-jaxws2.2.feature

```diff
@@ -1,22 +1,11 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=com.ibm.websphere.appserver.wsSecurity1.1-jaxws2.2
-IBM-App-ForceRestart: install, \
- uninstall
-IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.jaxws-2.2)))", \
+IBM-Provision-Capability:\
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.jaxws-2.2)))", \
```

Comment:
I believe this should be servlet 3, 3.1 and 4.0 and should have features listed of JAXWS-2.2 tolerates 2.3 in the feature list and com.ibm.websphere.appserver.wsSecurity1.1.internal.jaxws-2.2 and tolerate 2.3.

Reply:
Doing post this PR